src/app/zones/data/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { ZoneOverview, StatusKeyword } from '@/types/index.d'
+import { get } from '@/utilities/get'
+
+export function getZoneDpServerAuthType(zone: ZoneOverview): string {
+  const subscriptions = zone.zoneInsight?.subscriptions ?? []
+  if (subscriptions.length > 0) {
+    const lastSubscription = subscriptions[subscriptions.length - 1]
+    if (lastSubscription.config) {
+      const parsedConfig = JSON.parse(lastSubscription.config)
+      return get(parsedConfig, 'dpServer.auth.type', 'disabled')
+    }
+  }
+  return 'disabled'
```

Comment:
It's more `N/A` than disabled right?

Reply:
umm dunno, let me look this is another one that comes from previous code and I just moved it around a bit.

Reply:
I changed this to read `N/A` for the authenticationType if its empty/non-existent